database-mysql/dummyDataTenMillion.js

```diff
@@ -0,0 +1,102 @@
+//======================================
+//estimated time to completion: 30.32 minutes
+//======================================
```

Comment:
just popping in here, I really like this comment style.  super readable and breaks up the code really well